tensorflow_serving/model_servers/main.cc

```diff
@@ -346,7 +348,10 @@ int main(int argc, char** argv) {
                        "If non-empty, read an ascii PlatformConfigMap protobuf "
                        "from the supplied file name, and use that platform "
                        "config instead of the Tensorflow platform. (If used, "
-                       "--enable_batching is ignored.)")};
+                       "--enable_batching is ignored.)"),
+      tensorflow::Flag("per_process_gpu_memory_fraction", &per_process_gpu_memory_fraction,
```

Comment:
Please document that the default value is 0, which has special meaning and causes tensorflow to automatically select a value.

Reply:
have fixed~